tests/publiccloud/instance_overview.pm

```diff
@@ -46,11 +46,7 @@ sub run {
     assert_script_run("lsblk");
 
     # Install bzip2 to check for bsc#1165915
```

Comment:
maybe comment can be also removed ?

Reply:
My initial thought was that in case of error, it might help to lead the reviewer. As this seems to be the first zypper call after update.

Ofc, no objections for removal.

Reply:
ok , make sense